src/components/interactive-builder/modals/question/question-form/rendering-types/inputs/select/select-answers.component.tsx

```diff
@@ -152,6 +194,17 @@ const SelectAnswers: React.FC = () => {
         </div>
       )}
 
+      {/* Display an inline notification if any answer fails validation */}
+      {invalidAnswerFound && (
+        <InlineNotification
+          kind="error"
+          lowContrast
+          className={styles.error}
+          title={t('invalidAnswerConcept', 'Invalid Answer Concept Detected')}
+          subtitle={t('answerConceptValidation', 'One or more selected answer concepts do not exist in the system. ')}
+        />
+      )}
```

Comment:
This is good, maybe we can show a not found icon within the blue tag of the answer that is invalid. Otherwise we'd have to guess which ones are invalid.